GOVERNANCE.md

```diff
@@ -0,0 +1,125 @@
+# Diamond Bluff
+
+For an idea of what the goals of the Diamond Bluff project are, please see the
+[high level overview document](README.md).
+
+## Governance
+
+### Governance Goals
+
+The goal of governance for the Diamond Bluff project is to maintain and
+emphasize a technical meritocracy of contributors. Those who contribute the
+most and the best technical solutions, while supporting all contirbutors, have
+the most influence in the technical direction of the project. At the same
+time, a [Board of Directors](#Board of Directors) is established with it's
+own set of responsibilities, to help manage the Diamond Bluff project.
+
+## Membership
+
+The Diamond Bluff project is composed of Platinum, Silver, and Participating
```

Comment:
What about individuals? those who are not aligned with a company, nor with an academic institution. I'm not sure he will continue work with Diamond bluff, but Dave Schmitt is a prime example. He just retired from F5, but may want to be involved.